internal/v4/archive_test.go

```diff
@@ -352,6 +353,14 @@ func (s *ArchiveSuite) TestPostCharm(c *gc.C) {
 	s.assertUploadCharm(c, "POST", newResolvedURL("~charmers/precise/wordpress-1", -1), "mysql")
 }
 
+func (s *ArchiveSuite) TestPostCurrentVersion(c *gc.C) {
+	s.assertUploadCharm(c, "POST", newResolvedURL("~charmers/precise/wordpress-0", -1), "wordpress")
+
+	// Subsequent charm uploads should increment the
```

Comment:
is the comment correct?
it doesn't look like the revision number is increasing here.